unity/README.md

```diff
@@ -0,0 +1,17 @@
+# .NET Runtime - Unity Details
+
+This is Unity's fork of the .NET Runtime repository.
+
+The difference between this fork and the upstream repository should be as small as possible, with all of the differences specific to Unity. Our goal is to upstream as many changes made here as possible.
+
+## Pulling changes from upstream
+
+The [`main`](https://github.com/Unity-Technologies/runtime/tree/main) branch should always be in sync with the upstream [dotnet/runtime](https://github.com/dotnet/runtime) repository. There is a job in Unity's internal CI which runs weekly to pull the latest code from the upstream repo to this branch. The job will create a pull request to merge these changes to the [`unity-main`](https://github.cds.internal.unity3d.com/unity/il2cpp/pull/3602) branch.
```

Comment:
This link is wrong?  https://github.cds.internal.unity3d.com/unity/il2cpp/pull/3602

Reply:
Yes! I'll correct this.

Reply:
Fixed